quickannotator/populate_db.ipynb

```diff
@@ -214,16 +213,24 @@
     "        c = Point(np.random.randint(image_width), np.random.randint(image_height))\n",
     "        \n",
     "        poly: Polygon = generate_random_polygon(max_area=10000, centroid=(c.x, c.y))\n",
-    "        d = {\n",
-    "            \"image_id\": image_id,\n",
-    "            \"annotation_class_id\": annotation_class_id,\n",
-    "            \"isgt\": gtpred == \"gt\",\n",
-    "            \"centroid\": poly.centroid.wkt,  # Adding SRID=0 for pixel-based coordinates\n",
-    "            \"area\": poly.area,  # The area of the polygon\n",
-    "            \"polygon\": poly.wkt,  # Adding SRID=0 for the polygon\n",
-    "            \"custom_metrics\": json.dumps({\"iou\": 0.5})  # Convert custom_metrics to a JSON string\n",
-    "        }\n",
-    "        annotations.append(d)\n",
+    "        x = poly.centroid.x\n",
+    "        y = poly.centroid.y\n",
+    "        tiles = tiles_within_bbox(db, image_id, annotation_class_id, x, y, x, y)\n",
```

Comment:
this doesn't seem optimal. we *always* know the correct tile without having to do any query. i would as well suggest removing the GEOM polygon entirely from the tile annotation table. the correct tile id is:

```python
tile_row = query_col // tile_size
tile_col = query_row // tile_size
tileid = ntile_per_col*tile_row+ tile_col
```

this should remove the database queries and all downstream needs for any tile spatial query